README.md

```diff
@@ -1,24 +1,44 @@
-# Uplift iOS Client 
-[![GitHub release](https://img.shields.io/github/release/cuappdev/fitness-ios.svg)]()
-[![Platform](https://img.shields.io/badge/platform-ios-lightgrey.svg)]()
-[![GitHub contributors](https://img.shields.io/github/contributors/cuappdev/fitness-ios.svg)]()
-[![Build Status](https://travis-ci.org/cuappdev/clicker-ios.svg?branch=master)](https://travis-ci.org/cuappdev/fitness-ios)
+# Uplift - Cornell Fitness
 
-Fitness is one of the latest apps by [Cornell AppDev](http://cornellappdev.com), a project team at Cornell University.
+<p align="center"><img src=https://raw.githubusercontent.com/cuappdev/uplift-ios/master/Uplift/Assets.xcassets/AppIcon.appiconset/ItunesArtwork%402x.png width=210 /></p>
```

Comment:
Will we use the icon square for the image, or the app icon with curved edges?

Reply:
We're gonna use another image that has the icon plus the text "Uplift" next to it -- asked design for it but I need to upload it somewhere first